docs/en/Configuring-Alluxio-with-OrangeFS.md

```diff
@@ -9,11 +9,20 @@ priority: 2
 This guide describes how to configure Alluxio with [OrangeFS](http://www.orangefs.org/) as the 
 under storage system.
 
-# Initial Setup
-
-First, the Alluxio binaries must be on your machine. You can either
-[compile Alluxio](Building-Alluxio-Master-Branch.html), or
-[download the binaries locally](Running-Alluxio-Locally.html).
+# Building Alluxio-OrangeFS Distro
+
+The integration of Alluxio-OrangeFS depends on 
+[orangefs-jni](http://www.orangefs.org/fisheye/orangefs/browse/orangefs/branches/maven-repository/maven2/org/orangefs/usrint/orangefs-jni) 
+for an optimised I/O performance that is comparable to OrangeFS 
+[native performance](http://docs.orangefs.com/v_2_9/Design_Overview.htm).
```

Comment:
What about we do something like Spark, when jni is not available, display a warning message and uses a pure JVM implementationinstead?